### PR TITLE
Billing and Metring for images- BM3PI

### PR DESCRIPTION
**Pull request by sreekarbvibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by sreekarbvibm** [comment]
[billing_metering_ga_testing.log.txt](https://github.com/user-attachments/files/15764936/billing_metering_ga_testing.log.txt)

**Comment by ujjwal-ibm** [comment]
This PR needs #5407 to be merged first

File: scripts/build.sh [hunk 1]
```diff
@@ -51,13 +51,14 @@ if [[ -n "${TF_RELEASE}" ]]; then
...
 # Build!
+VERSION="1.64.0"
 echo "==> Building..."
 gox \
     -os="${XC_OS}" \
     -arch="${XC_ARCH}" \
     -osarch="${XC_EXCLUDE_OSARCH}" \
     -ldflags "${LD_FLAGS}" \
-    -output "pkg/{{.OS}}_{{.Arch}}/terraform-provider-ibm" \
+    -output "pkg/{{.OS}}_{{.Arch}}/terraform-provider-ibm_${VERSION}_{{.OS}}_{{.Arch}}" \
```
Comment (hkantare) [inline scripts/build.sh:61]: These changes are required?

Comment (sreekarbvibm) [reply scripts/build.sh:61]: No. Let me remove.

Comment (sreekarbvibm) [reply scripts/build.sh]: @ujjwal-ibm I have removed the all changes in this file.

**Pull request merged**